utils/run_test.sh

```diff
@@ -274,9 +274,11 @@ if [ -d "/mnt/daos" ]; then
     run_test src/common/tests/btree.sh dyn -s ${BTREE_SIZE}
 
     COMP="UTEST_csum"
-    run_test "${SL_PREFIX}/bin/srv_checksum_tests"
-    run_test "${SL_PREFIX}/bin/pool_scrubbing_tests"
-    run_test "${SL_PREFIX}/bin/rpc_tests"
+    /home/rjensen1/workspace/daos/src/object/tests/cli_checksum_tests.c
```

Comment:
You cannot have a hard-coded path like this.  You also cannot execute `.c` files.  I'm assuming this was some kind of copy'n'pasto mistake.